ERROR - 2024-11-29 08:36:19 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'francisco_db', 3306, '', 0)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\Padayon\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Padayon\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Padayon\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(35): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->add()
#9 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#10 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#3 C:\xampp\htdocs\Padayon\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\Padayon\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\Padayon\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(35): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->add()
#8 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#9 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-11-29 08:36:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'francisco_db'
[Method: POST, Route: users/add]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tblusers` (`username`, `password`, `email`, `fullname`) VALUES (:username:, :password:, :email:, :fullname:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Users.php(35): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->add()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-29 09:19:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-29 09:19:19 --> [DEPRECATED] Creation of dynamic property App\Controllers\Users::$session is deprecated in APPPATH\Controllers\Users.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Users->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-11-29 09:19:19 --> [DEPRECATED] Creation of dynamic property App\Controllers\Users::$usersModel is deprecated in APPPATH\Controllers\Users.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Users->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 09:19:20 --> ErrorException: Attempt to read property "id" on array
[Method: GET, Route: users]
in APPPATH\Views\users_view.php on line 19.
 1 APPPATH\Views\users_view.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "id" on array', 'C:\\xampp\\htdocs\\Padayon\\app\\Views\\users_view.php', 19)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Padayon\\app\\Views\\users_view.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('users_view', [], true)
 5 APPPATH\Controllers\Users.php(24): view('users_view', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-29 09:19:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-29 09:19:33 --> [DEPRECATED] Creation of dynamic property App\Controllers\Users::$session is deprecated in APPPATH\Controllers\Users.php on line 12.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Users->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-11-29 09:19:33 --> [DEPRECATED] Creation of dynamic property App\Controllers\Users::$usersModel is deprecated in APPPATH\Controllers\Users.php on line 13.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Users->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-29 09:46:15 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'francisco_db', 3306, '', 0)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(10): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->index()
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(10): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->index()
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-29 09:46:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'francisco_db'
[Method: GET, Route: users]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tblusers`', [], false)
 3 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Users.php(10): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-29 09:47:38 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'francisco_db', 3306, '', 0)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(10): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->index()
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#8 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Padayon\app\Controllers\Users.php(10): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Users->index()
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-29 09:47:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'francisco_db'
[Method: GET, Route: users]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tblusers`', [], false)
 3 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Users.php(10): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-29 09:52:39 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'francisco_db', 3306, '', 0)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Padayon\app\Controllers\Products.php(10): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Products->index()
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'francisco_db' in C:\xampp\htdocs\Padayon\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Padayon\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Padayon\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Padayon\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Padayon\app\Controllers\Products.php(10): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(933): App\Controllers\Products->index()
#6 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#7 C:\xampp\htdocs\Padayon\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\Padayon\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\Padayon\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\Padayon\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-29 09:52:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'francisco_db'
[Method: GET, Route: products]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `producttbl`', [], false)
 3 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Products.php(10): CodeIgniter\Model->__call('get', [])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Products->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:06:18 --> ErrorException: Undefined property: stdClass::$active
[Method: GET, Route: users]
in APPPATH\Views\users_view.php on line 26.
 1 APPPATH\Views\users_view.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$active', 'C:\\xampp\\htdocs\\Padayon\\app\\Views\\users_view.php', 26)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\Padayon\\app\\Views\\users_view.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('users_view', [], true)
 5 APPPATH\Controllers\Users.php(18): view('users_view', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Users->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Users))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:28:47 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: home]
in APPPATH\Controllers\Index.php on line 70.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:28:56 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: home]
in APPPATH\Controllers\Index.php on line 70.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:28:57 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: home]
in APPPATH\Controllers\Index.php on line 70.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-29 10:34:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:34:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:35:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:36:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-29 10:37:27 --> ErrorException: Undefined variable $user
[Method: GET, Route: /]
in APPPATH\Controllers\Index.php on line 31.
 1 APPPATH\Controllers\Index.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php', 31)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Index->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Index))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:39:09 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: /]
in APPPATH\Controllers\Index.php on line 75.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:39:10 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: /]
in APPPATH\Controllers\Index.php on line 75.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:39:11 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: home]
in APPPATH\Controllers\Index.php on line 75.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-29 10:39:16 --> ParseError: syntax error, unexpected variable "$data", expecting "function" or "const"
[Method: GET, Route: /]
in APPPATH\Controllers\Index.php on line 75.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Index')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Index')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\Index', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-29 10:39:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:39:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:40:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-29 10:40:33 --> ErrorException: Undefined variable $user
[Method: POST, Route: index/login]
in APPPATH\Controllers\Index.php on line 33.
 1 APPPATH\Controllers\Index.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\Padayon\\app\\Controllers\\Index.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Index->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Index))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\Index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-11-29 10:41:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:41:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:41:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:41:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-29 10:41:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
